_scripts/app1-4.R

```diff
@@ -1,31 +1,31 @@
 {
-  rm(list=ls())                         # clean out the environment
-  options(show.error.locations = TRUE)  # give line numbers on errors
+  rm(list=ls());                       # clean out the environment
+  options(show.error.locations = TRUE);  # give line numbers on errors
   
-  fish1Weight = 45
-  fish2Weight = 64
-  fish3Weight = 50
-  fish4Weight = 58
-  fish5Weight = 49
+  fish1Weight <- 45;
+  fish2Weight <- 64;
+  fish3Weight <- 50;
+  fish4Weight <- 58;
+  fish5Weight <- 49;
   
   # solve for the mean fish weight
-  meanWeight = fish1Weight + fish2Weight + Fish3weight + fish4Weight + fish5Weight /5
+  meanWeight <- (fish1Weight + fish2Weight + fish3Weight + fish4Weight + fish5Weight) /5;
   
   # solve the variance for each fish
-  fish1Var = (fish1Weight - meanWeight)^2
-  fish2Var = (fish1Weight - meanWeight)^2
-  fish3Var = (fish1Weight - meanWeight)^2
-  fish4Var = (fish1Weight - meanWeihgt)^2
-  fish5Var = (fish1Weight - meanWeight)^2
+  fish1Var <- (fish1Weight - meanWeight)^2;
```

Comment:
I do not care if you use <- or = in you project.  I prefer = because it is almost universal for programming languages -- and it requires one less keystroke!

In R, there is also the -> operator so you can say 3 -> a and it will set a to 3 (as opposed to a <- 3) but I almost never see this used anymore.